podcasts/Whats New/Announcements.swift

```diff
@@ -1,15 +1,30 @@
 import Foundation
+import SwiftUI
 
 struct Announcements {
     // Order is important.
     // In the case a user migrates to, let's say, 7.10 to 7.15 and
     // there were two announcements, the last one will be picked.
     var announcements: [WhatsNew.Announcement] = [
         .init(
-            version: 7.80,
-            image: "",
-            title: "",
-            message: ""
+            version: 7.43,
+            header: {
+                AnyView(AutoplayWhatsNewHeader())
```

Comment:
I know using `AnyView` is bad, but I couldn't think of any other solution. :(

Any input is appreciated. :)